<!-- -*- mode: XML -*- -->
<launch>
    <arg name="connect_on_startup"   default="true"/>
    <arg name="calibrate_on_startup" default="false"/>
    <arg name="engage_on_startup"    default="true"/>
	<arg name="simulation_mode"      default="true"/>
	
    
    <arg name="output"   default="screen"/>
    <arg name="respawn"   default="false"/>
    
    <node pkg="boris_gimbal" type="boris_gimbal_node" name="odrive_node" ns="odrive_node" 
            output="$(arg output)" respawn="$(arg respawn)">
        
        <param name="connect_on_startup"   value="$(arg connect_on_startup)"/>
        <param name="calibrate_on_startup" value="$(arg calibrate_on_startup)"/>
        <param name="engage_on_startup"    value="$(arg engage_on_startup)"/>
		<param name="simulation_mode"      value="$(arg simulation_mode)"/>
        
    </node>
</launch>
